doc/user/pimv6.rst

```diff
@@ -113,6 +113,11 @@ Certain signals have special meanings to *pim6d*.
    notifications to the kernel. This command is vrf aware, to configure for a
    vrf, enter the vrf submode.
 
+.. clicmd:: ipv6 ssmpingd X:X::X:X
+
+   Enable ipv6 ssmpingd configuration. The IPv6 address is a unicast address
```

Comment:
Can you please elaborate the usage of this command.

Reply:
Ack